components/dashboard/src/components/Modal.tsx

```diff
@@ -143,11 +149,18 @@ const MaybeWithForm: FC<MaybeWithFormProps> = ({ onSubmit, disabled, children })
     );
 
     if (!onSubmit) {
-        return <>{children}</>;
+        return (
+            <div className="flex flex-col max-h-[calc(100dvh-3rem)] min-[576px]:max-h-[calc(100dvh-6rem)]">
```

Comment:
This is duplicating the classes below, but I don't really like storing utility classes in variables, which don't offer any autocomplete or other language features and can lead to confusion about what styles actually apply. Took inspiration from https://tailwindcss.com/docs/reusing-styles here.